client/src/actions/bulk.js

```diff
@@ -34,16 +34,17 @@ function buildFormData(form) {
     return formData;
 }
 
-function doAction(action, form, isAsyncJob) {
+function doAction(action, form) {
     return async dispatch => {
         dispatch(pending(action));
         try {
-            let result = await (await fetch(action.uri + (isAsyncJob ? '/start' : ''), {
+            let result = await (await fetch(action.uri, {
                 method: 'post',
                 body: buildFormData(form)
             })).json();
 
-            if (isAsyncJob) {
+            // TODO: Use this logic when approving a BulkCreateMonumentSuggestion
+            /*if (isAsyncJob) {
```

Comment:
@njd9419 Is inserting a bunch of suggestions not slow? I feel like we still may need this to use AsyncJobs

Reply:
That's a good point that I hadn't considered. I haven't tested it but we have the capability to do it via `AsyncJob` so we might as well. I'll do that in the Suggestions UI PR since I'm going to be doing it for `BulkCreateMonumentSuggestion` approval then too.